website/content/docs/v0.9/Guides/disk-encryption.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Disk Encryption"
+description: "This page contains guides related kernel level disk encryption"
+---
+
```

Comment:
I think we should start with the overview - what gets encrypted, what is not encrypted and why